lib/unifex/native_code_generator.ex

```diff
@@ -99,8 +87,33 @@ defmodule Unifex.NativeCodeGenerator do
     content
   end
 
-  defp indent(line) do
-    "  #{line}"
+  @doc """
+  Helper for generating code. Uses `sigil_g/2` underneath.
+
+  It supports all the flags supported by `sigil_g/2` and the following ones:
+  * `j(joiner)` - joins list of strings using `joiner`
+  * n - alias for `j(\\n)`
+
+  If passed a list and flags supported by `sigil_g/2`, each flag will be executed
+  on each element of the list, until the list is joined by using `j` or `n` flag.
+  """
+  @spec g(String.Chars.t() | [String.Chars.t()], charlist()) :: String.t() | [String.t()]
+  def g(content, 'j(' ++ flags) when is_list(content) do
```

Comment:
Please, rename it to something meaningful like `formatter`. Even `gen` would be better.